gulpfile.ts

```diff
@@ -73,3 +74,27 @@ gulp.task(
 gulp.task('default', gulp.series('cjs', 'esm', 'umd', 'umd:min'));
 
 gulp.task('isolate', gulp.series(createIsolateTasksFromConfig(isolateBuildConfig)));
+
+const project: { ref?: gulpTs.Project } = { };
+
+gulp.task(
+  'dev',
+  gulp.series(
+    function build() {
+      return gulpFactory(
+        { globs: 'src/**/*.ts' },
+        'dist/dev',
+        {
+          module: 'ES2015',
+          declaration: true
+        },
+        true,
+        false,
+        project
```

Comment:
Why not `{}`?

Reply:
This is the variable that for incremental build.

Once a build is performed, the build information is stored in the `project`, and the next build is performed faster.

Reply:
I think `{}` is equal to `project` because of the feature of js closure?

Reply:
I just tried, but it doesn't work.

Reply:
All right, but it doesn't make sense.